### ISSUE TITLE
Tracker: RevolutionTT

### ISSUE THREAD

**Issue by ps2wiz** [SickChill/SickChill#4222]
TLDR` = Read my last paragraph

I'm following instructions from the sickrage forum, request section to post all new request on GitHub. But, the link posted didn't work anymore, so I hope I am in the right section. I apologize in advance if I am not, as this is more of a suggestion & not an issue.

I have searched Google and Sickrage forums extensively, about the tracker RevolutionTT.me (private torrent tracker).

A few years ago it was suggested in a long "tracker suggestion..." thread, on sickrage forums which is now closed without a reply. RevolutionTT.me  hasn't been added or mentioned much besides on sickbeard (which looks like it supported it a few years ago - don't if it still does). I don't use sickbeard so I can't verify.

So my request is simple, I would love to see RevolutionTT.me as search provider (a private torrent tracker) for my favorite 'automatic video manager of TV shows'. Since it's a private site, I can provide access to a developer or anyone else that can help.

Thanks in advance.

**Comment by bobbysteel** [comment]
in the short term use Jackett to proxy the results into Sickrage
